Makefile

```diff
@@ -85,13 +99,23 @@ build: # Build REANA client and cluster components.
 	reana-dev install-client && \
 	reana-dev install-cluster && \
 	reana-dev git-submodule --update && \
-	reana-dev docker-build -t latest && \
-	reana-dev git-submodule --delete
+	reana-dev docker-build -b DEBUG=${DEBUG} && \
+	reana-dev git-submodule --delete && \
+	if [ "${DEBUG}" -gt 0 ]; then \
+		echo ${ccyellow} && \
+		echo "Please run minikube mount in a new terminal to have live code updates." && \
+		echo "minikube mount $$(pwd)/..:/code" && \
+		echo "For more information visit the documentation: \nhttps://reana-cluster.readthedocs.io/en/latest/developerguide.html#deploying-latest-master-branch-versions" && \
+		echo ${ccend}; \
+	fi
 
 deploy: # Deploy/redeploy previously built REANA cluster.
+ifeq ($(SHOULD_MINIKUBE_MOUNT),1)
+	$(error "Looks like you are not running minikube mount. Please run the following command on a different session: minikube mount $$(pwd)/..:/code")
```

Comment:
Separate command to its own line, so that the output would look like:

```
It seems you are not running 'minikube mount'.  Please run the following command in a different terminal:

$ minikube mount ...

This will enable the cluster pods to see the live edits that are necessary for debugging.

Exiting.
```

```